tests/e2e/labelupdates.go

```diff
@@ -467,12 +467,9 @@ var _ bool = ginkgo.Describe("[csi-block-vanilla] [csi-block-vanilla-parallelize
 		ginkgo.By(fmt.Sprintf("Deleting pvc %s in namespace %s", pvc.Name, pvc.Namespace))
 		err = client.CoreV1().PersistentVolumeClaims(namespace).Delete(ctx, pvc.Name, *metav1.NewDeleteOptions(0))
 		gomega.Expect(err).NotTo(gomega.HaveOccurred())
-		// Waiting for some time for PVC to be deleted correctly
-		ginkgo.By(fmt.Sprintf("Sleeping for %v seconds to allow PVC deletion", oneMinuteWaitTimeInSeconds))
-		time.Sleep(time.Duration(oneMinuteWaitTimeInSeconds) * time.Second)
-
-		_, err = e2eVSphere.getLabelsForCNSVolume(pv.Spec.CSI.VolumeHandle,
```

Comment:
we should be verifying PVC entry is deleted in cns volume, why was this  removed

Reply:
there was delay in one of the setup for PVC deletion for the allocated oneMinuteWaitTimeInSeconds. So used polling mechanism instead of static wait for pvc deletion and then validating based on returned err value